ui/js/directives/treeherder/clonejobs.js

```diff
@@ -900,7 +898,10 @@ treeherder.directive('thCloneJobs', [
         };
 
         var generateJobElements = function(resultsetAggregateId, resultset) {
-
+            ReactDOM.render(
+              React.createElement(window.Hello, {name: "Let's ReactJS!"}),
+              document.getElementsByClassName("job-list")[0]
```

Comment:
for the resultset table you have an id, so yes, I believe that's the right function to use